src/dvclive/live.py

```diff
@@ -299,6 +307,33 @@ def log_param(
         """Saves the given parameter value to yaml"""
         self.log_params({name: val})
 
+    def log_artifact(
+        self,
+        path: str,
+    ):
+        from dvc.exceptions import OutputDuplicationError
+
+        """Saves a local file or directory as an artifact"""
+        if not isinstance(path, str):
+            raise InvalidDataTypeError("artifact", type(path))
+
+        if self._dvc_repo is not None:
+            if path in self._outs or self._inside_dvc_exp:
+                stage = self._dvc_repo.commit(path, force=True)
```

Comment:
Giving it a second thought. Do we really need to call `commit`? I think `add` will basically end in the same result